src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -209,7 +209,7 @@ public bool ReturnsByRefReadonly
         /// </summary>
         public TypeSymbol ReturnType => ReturnTypeWithAnnotations.Type;
 
-        public abstract FlowAnalysisAnnotations ReturnTypeAnnotationAttributes { get; }
+        public abstract FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations { get; }
```

Comment:
>ReturnTypeFlowAnalysisAnnotations [](start = 48, length = 33)

Minor: The rename could probably be done in a separate PR.

Reply:
I prefer to keep the rename in this PR. Otherwise I fear it will never get adjusted (will forget, other priorities, etc).

---
In reply to: [295432948](https://github.com/dotnet/roslyn/pull/36444#discussion_r295432948) [](ancestors = 295432948)